src/privacy-policy.md.njk

```diff
@@ -5,52 +5,105 @@ layout: layout-single-page-prose.njk
+The legal basis for processing this data is that processing is necessary for the performance of a task carried out in the public interest or in the exercise of official authority vested in the controller. This is because the GOV.UK Design System brings together the latest research, design and development from across government to make sure it’s representative and relevant for its users.
 
-We will not share your information with any other organisations for marketing, market research or commercial purposes, and we do not pass on your details to other websites.
+## How long we keep your data
+
+We will only keep your personal data for as long as:
+- the law requires us to
+- we need for the purposes listed above
+
+This means that we will only hold your personal data for a minimum of 1 year and a maximum of 7 years.
+
+If you have signed up to our mailing list, your email address will only be retained while you choose to remain on the list.
+
+## Where your data is processed and stored
+
+We design, build and run our systems to make sure that your data is as safe as possible at any stage, both while it’s processed and when it’s stored. 
+
+Your personal data may be transferred outside the United Kingdom while being processed by GDS. If this happens, we’ll make sure you’re given the same level of technical and legal protection as you are within the United Kingdom.
+
+## Who we share your data with 
+
+As part of [GOV.UK Design System](https://design-system.service.gov.uk/) we share your personal data with data processors who provide us with:
+- software collaboration platforms when you share research, feedback or make a contribution
+- mailing list providers when you sign up to receive emails from us
+- support providers when you contact us for assistance
+
+We will not:
+- sell or rent your data to third parties
+- share your data with third parties for marketing purposes
+
+We will share your data if we’re required to do so by law — for example, by court order, or to prevent fraud or other crime.
+
+## How we protect your data and keep it secure
+We are committed to doing all that we can to keep your data secure. We set up systems and processes to prevent unauthorised access to or disclosure of the data 
+we collect about you – for example, we protect your data using varying levels of encryption. All third parties that process personal data for GDS are required to keep that data secure.
 
 ## Your rights
 
-You can [find out what information we hold about you](https://www.gov.uk/data-protection), and [ask us not to use any of the information we collect](https://www.gov.uk/contact).
+You have the right to request:
+- information about how your personal data is processed, and to request a copy of that personal data
+- that any inaccuracies in your personal data is rectified without delay
```

Comment:
I think this should be "are rectified", as inaccuracies is plural? Happy to be wrong though 😁

Reply:
Nope you're right, inaccuracies is plural.
```suggestion
- that any inaccuracies in your personal data are rectified without delay
```